runtime/lua/vim/_editor.lua

```diff
@@ -329,7 +329,13 @@ do
       vim.api.nvim_command(('echo "%s"'):format(dots))
     end
     if is_last_chunk then
+      if startpos then
+        vim.fn.setpos("'[", startpos)
+        startpos = nil
+      end
       vim.api.nvim_command('redraw' .. (tick > 1 and '|echo ""' or ''))
+    elseif startpos == nil then
+      startpos = vim.fn.getpos("'[")
```

Comment:
```suggestion
    elseif startpos == 1 then
      startpos = vim.fn.getpos("'[")
```

Reply:
This will not work if the first chunk is empty